utils/scale-test

```diff
@@ -269,6 +275,8 @@ def main():
 
     args = parser.parse_args(sys.argv[1:])
     (rng, runs) = run_many(args)
+    if args.tmpdir != None:
+        shutil.rmtree(args.tmpdir)
```

Comment:
Eh, on second thought some users might not expect a tmpdir to be deleted before use, I'll just remove these lines.